Dockerfile-debian.template

```diff
@@ -114,13 +114,12 @@ RUN set -eux; \
 	; \
 	rm -rf /var/lib/apt/lists/*; \
 	\
-	gosu redmine bundle install --jobs "$(nproc)" --without development test; \
+	echo > ./config/database.yml; \
```

Comment:
```suggestion
# fill up "database.yml" with bogus entries so the redmine Gemfile will pre-install all database adapter dependencies
# https://github.com/redmine/redmine/blob/e9f9767089a4e3efbd73c35fc55c5c7eb85dd7d3/Gemfile#L50-L79
	echo '# the following entries only exist to force `bundle install` to pre-install all database adapter dependencies -- they can be safely removed/ignored' > ./config/database.yml; \
```

Reply:
the database.yml is removed in the same layer :)